sdk/monitor/azure-monitor-ingestion/azure/monitor/ingestion/_operations/_operations.py

```diff
@@ -82,81 +82,6 @@ def build_logs_ingestion_upload_request(
 
 
 class LogsIngestionClientOperationsMixin(LogsIngestionClientMixinABC):
-    @overload
```

Comment:
Yep, revived the autorest.python issue linked above and have engaged the autorest team (i.e. Izzy) about a way to have autorest not generated these overloads for specific operations.

Reply:
Seems the plan is to add an "internal" directive which will allow us to generate ops as internal if we want to override and change a lot. This will allow more flexibility in determining what is exported to users.

Reply:
Awesome thanks! Do we think this new process will be in place for the upcoming release? Or something to onboard to at a later date?

Reply:
Definitely something that will be onboarded at a later date since release is this week, and hopefully by the next time I regenerate. 🤞